opencga-app/app/scripts/azure/arm/avere/mount.py

```diff
@@ -0,0 +1,201 @@
+import sys
+import socket
+import fcntl
+import struct
+import random
+import os
+import shutil
+import subprocess
+import time
+
+# Run `python -m unittest discover` in this dir to execute tests
+
+default_mount_options_nfs = "nfs hard,nointr,proto=tcp,mountproto=tcp,retry=30 0 0"
+default_mount_options_cifs = "dir_mode=0777,file_mode=0777,serverino,nofail,vers=3.0"
+
+
+def get_avere_ips(vserver_string):
+    vserver_range = vserver_string.split("-")
+    if len(vserver_range) < 2:
+        print("Expect vserver ip range to be in the format 'startip-endip' got:" + vserver_string)
+        exit(3)
+
+    all_ips = []
+    start_ip = vserver_range[0]
+    end_ip = vserver_range[1]
+
+    start_ip_parts = start_ip.split(".")
+    start_ip_last_digit = int(start_ip_parts[3])
+    ip_prefix = start_ip_parts[0] + "." + start_ip_parts[1] + "." + start_ip_parts[2] + "."
+    end_ip_last_digit = int(end_ip.split(".")[3])
+
+    for ip in range(start_ip_last_digit, end_ip_last_digit + 1): 
+        all_ips.append(ip_prefix + str(ip))
+    
+    return all_ips
+
+def get_ip_address():
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    try:
+        # doesn't even have to be reachable
+        s.connect(('10.255.255.255', 1))
+        IP = s.getsockname()[0]
+    except:
+        IP = '127.0.0.1'
+    finally:
+        s.close()
+    return IP
+
+def ip_as_int(ip):
+    o = map(int, ip.split('.'))
+    res = (16777216 * o[0]) + (65536 * o[1]) + (256 * o[2]) + o[3]
+    return res
+
+def remove_lines_containing(file, contains):
+    with open(file,"r+") as file:
+        d = file.readlines()
+        file.seek(0)
+        for i in d:
+            if contains not in i and i != "\n":
+                file.write(i)
+        file.truncate()
+
+def print_help():
+    print "For example 'sudo python mount.py avere 10.0.1.10-10.0.1.14'"
+    print "or 'sudo python mount.py azurefiles <storage-account-name>,<share-name>,<storage-account-key>'"
+
+def install_apt_package(package):
+    try:
+        print "Attempt to install " + package
+        subprocess.check_call(["apt", "install", package, "-y"])
+        print "Install completed successfully"
+    except subprocess.CalledProcessError as e:
+        print "Failed install " + package + " error:" + e.message
+        exit(4)
+
+def main():  
```

Comment:
`main` is pretty lengthy

Reply:
agreed, refactored into azurefiles and avere methods